2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,53 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+  let recipe1 = {
+    title: 'Mole',
+    servings: 2,
+    ingredients:['cinnamon','cumin','cocoa']
+  }
+  console.log(recipe1.title);
+  console.log(recipe1.servings);
+  recipe1.ingredients.forEach(element=>console.log(element));
```

Comment:
This is a very small comment, but it is often better to have variable names that represent the data we have. Here you could have used:

`recipe1.ingredients.forEach(ingredient=>console.log(ingredient));`